### ISSUE TITLE
Missing json payload for EmbeddedRecordsMixin

### ISSUE THREAD

**Issue by benoittgt** [emberjs/guides#791]
Hello

It seems we are missing json payload when explaining [EmbeddedRecordsMixin](http://guides.emberjs.com/v2.0.0/models/customizing-serializers/#toc_embeddedrecordmixin)

Atfer "For example if your post model contained an embedded author record" we should add json as used in other parts of the guide like  " the JSONSerializer expects the response to be a JSON object that looks similar to this:"

Maybe with a json payload like:

```json
{
    "id": 1,
    "title": "Rails is omakase",
    "tag": "rails",
    "authors": [
        {
            "id": 2,
            "name": "Steve"
        }
    ]
}
```

I'm maybe wrong for this json, if yes this is for me a good reason to add json payload to the guide. I can make a PR if the json is ok.

**Comment by locks** [comment]
That JSON is incorrect, it should be in the [json:api format](http://jsonapi.org/).

**Comment by benoittgt** [comment]
Hello @locks. Hope everything is going good since RubyConfpt :smile: 

For the json you're right maybe something like that. What do you think about it @bmac ?

I understand now why it's not in the documentation...

```json
{
   "data":[
      {
         "type":"articles",
         "id":"1",
         "attributes":{
            "title":"I made a trip"
         },
         "links":{
            "self":"http://example.com/articles/1"
         },
         "relationships":{
            "comments":{
               "links":{
                  "self":"http://example.com/articles/1/relationships/comments",
                  "related":"http://example.com/articles/1/comments"
               },
               "data":[
                  {
                     "type":"comments",
                     "id":"5"
                  },
                  {
                     "type":"comments",
                     "id":"12"
                  }
               ]
            }
         }
      },
      {
         "type":"articles",
         "id":"2",
         "attributes":{
            "title":"I made a trip again"
         },
         "links":{
            "self":"http://example.com/articles/2"
         },
         "relationships":{
            "comments":{
               "links":{
                  "self":"http://example.com/articles/2/relationships/comments",
                  "related":"http://example.com/articles/2/comments"
               },
               "data":[
                  {
                     "type":"comments",
                     "id":"6"
                  },
                  {
                     "type":"comments",
                     "id":"13"
                  }
               ]
            }
         }
      }
   ],
   "included":[
      {
         "type":"comments",
         "id":"5",
         "attributes":{
            "body":"Nice trip!"
         },
         "relationships":{
            "author":{
               "data":{
                  "type":"people",
                  "id":"2"
               }
            }
         },
         "links":{
            "self":"http://example.com/comments/5"
         }
      },
      {
         "type":"comments",
         "id":"12",
         "attributes":{
            "body":"I like Portugal"
         },
         "relationships":{
            "author":{
               "data":{
                  "type":"people",
                  "id":"9"
               }
            }
         },
         "links":{
            "self":"http://example.com/comments/12"
         }
      },
      {
         "type":"comments",
         "id":"6",
         "attributes":{
            "body":"I like your blog post"
         },
         "relationships":{
            "author":{
               "data":{
                  "type":"people",
                  "id":"8"
               }
            }
         },
         "links":{
            "self":"http://example.com/comments/6"
         }
      },
      {
         "type":"comments",
         "id":"13",
         "attributes":{
            "body":"I like Portugal and Belgium"
         },
         "relationships":{
            "author":{
               "data":{
                  "type":"people",
                  "id":"10"
               }
            }
         },
         "links":{
            "self":"http://example.com/comments/13"
         }
      }
   ]
}
```

**Comment by bmac** [comment]
Currently the `EmbeddedRecordsMixin` does not support working with the `JSONAPISerializer`. The guides code example @benoittgt linked shows the `EmbeddedRecordsMixin` being combined with the `JSONSerializer` so I think his original json payload is correct.

**Comment by benoittgt** [comment]
If I understand properly what you said @bmac I made a PR with [the initial json](https://github.com/emberjs/guides/issues/791#issue-107953194)

**Comment by locks** [comment]
Fixed by #803.
